tests/conftest.py

```diff
@@ -469,6 +469,29 @@ async def f() -> List[str]:
     return f
 
 
+@pytest.fixture(scope="function")
+def get_type_of_column(postgresql_client) -> Callable[[], Awaitable[List[str]]]:
```

Comment:
Sorry. I overlooked this one.

```suggestion
def get_type_of_column(postgresql_client) -> Callable[[], Awaitable[Optional[str]]]:
```